docs/developers/set-metrics.md

```diff
@@ -1,3 +1,63 @@
 # Monitoring Application
 
-> TODO
\ No newline at end of file
+If your application has exposed metrics, you can easily setup monitoring system
+with the help of `metric` capability.
+
+Let's run [`christianhxc/gorandom:1.0`](https://github.com/christianhxc/prometheus-tutorial) as an example app.
+The app will emit random latencies as metrics.
+
+```console
+$ vela svc deploy metricapp -t webservice --image christianhxc/gorandom:1.0 --port 8080
+```
+
+Then add metric by:
+
+```console
+$ vela metric metricapp
+Adding metric for app metricapp
+⠋ Deploying ...
+✅ Application Deployed Successfully!
+Showing status of service(type: webservice) metricapp deployed in Environment myenv
+Service metricapp Status:	 HEALTHY Ready: 1/1
+	metric: scheme=http
```

Comment:
Could you check the markdown view? It is not aligned well:

```
Service metricapp Status:	 HEALTHY Ready: 1/1
	metric: scheme=http
        enabled=true
        format=prometheus
        path=/metrics
```

Reply:
yes, but I'm just copy-paste output. It's hard to tuning the formal well